styles/css/core/variables.css

```diff
@@ -36,26 +36,26 @@
   --pgn-transition-base: all .2s ease-in-out; /* Generic transition for any property change */
   --pgn-transition-progress-bar-bar-transition: width .6s ease;
   --pgn-transition-progress-bar-bar-animation-timing: 1s linear infinite;
-  --pgn-transition-form-control: background-color .15s ease-in-out, border-color .15s ease-in-out, box-shadow .15s ease-in-out;
-  --pgn-transition-form-input: border-color .15s ease-in-out, box-shadow .15s ease-in-out;
+  --pgn-transition-form-control: background-color 0.15s ease-in-out, border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+  --pgn-transition-form-input: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
   --pgn-transition-carousel-control: opacity .15s ease;
   --pgn-transition-carousel-indicator: opacity .6s ease;
-  --pgn-transition-carousel-duration: .6s;
+  --pgn-transition-carousel-duration: 0.6s;
   --pgn-transition-carousel-base: transform var(--pgn-transition-carousel-duration) ease-in-out;
   --pgn-transition-btn: none;
   --pgn-transition-badge: none;
   --pgn-typography-print-page-size: a3;
   --pgn-typography-line-height-display-mobile: 3.5rem; /* Mobile display line height. */
   --pgn-typography-line-height-display-base: 1; /* Standard display line height. */
-  --pgn-typography-line-height-micro: .938rem; /* Micro utils line height. */
+  --pgn-typography-line-height-micro: 0.938rem; /* Micro utils line height. */
   --pgn-typography-line-height-sm: 1.5; /* Small line height. */
   --pgn-typography-line-height-lg: 1.5; /* Large line height. */
   --pgn-typography-line-height-base: 1.5556; /* Basic line height. */
   --pgn-typography-font-weight-display-4: var(--pgn-typography-font-weight-bold); /* Font weight of display level 4. */
   --pgn-typography-font-weight-display-3: var(--pgn-typography-font-weight-bold); /* Font weight of display level 3. */
   --pgn-typography-font-weight-display-2: var(--pgn-typography-font-weight-bold); /* Font weight of display level 2. */
   --pgn-typography-font-weight-display-1: var(--pgn-typography-font-weight-bold); /* Font weight of display level 1. */
-  --pgn-typography-font-weight-table-th: bold; /* Table th font weight. */
+  --pgn-typography-font-weight-table-th: 700; /* Table th font weight. */
```

Comment:
[inform] `sd-transforms` changes `bold` to the corresponding, explicit font weight.